install/kubernetes/helm/istio/values.yaml

```diff
@@ -136,19 +147,19 @@ global:
     # Resources for the sidecar.
     resources:
       requests:
-        cpu: 10m
-      #  memory: 128Mi
-      # limits:
-      #   cpu: 100m
-      #   memory: 128Mi
+        cpu: 100m
+        memory: 128Mi
+      limits:
+        cpu: 2000m
+        memory: 128Mi
 
     # Controls number of Proxy worker threads.
     # If set to 0 (default), then start worker thread for each CPU thread/core.
     concurrency: 0
 
     # Configures the access log for each sidecar. Setting it to an empty string will
     # disable access log for sidecar.
-    accessLogFile: "/dev/stdout"
+    accessLogFile: ""
```

Comment:
Can you update the comment - 'set it to /dev/stdout to generate logs in the envoy container' ?

Reply:
done